src/simulators/matrix_product_state/matrix_product_state_internal.cpp

```diff
@@ -1178,6 +1178,23 @@ uint_t binary_search(const rvector_t &acc_probvector,
     return binary_search(acc_probvector, mid, end, rnd);
 }
 
+double MPS:: norm(const uint_t qubit, const cvector_t &vmat) const {
```

Comment:
Is this function necessary if there are no specific optimizations for single-qubit?

Reply:
I agree. Removed.